backend/core/src/migration/1641855882656-remove-application-due-time.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class removeApplicationDueTime1641855882656 implements MigrationInterface {
+  name = "removeApplicationDueTime1641855882656"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const listings = await queryRunner.query(
+      `SELECT id, application_due_time, application_due_date from listings`
+    )
+
+    for (const listing of listings) {
```

Comment:
@seanmalbert @YazeedLoonat Made the updates here to put existing time fields onto the date fields as we remove the time, but I struggled a little here so would love some eyes on this new piece before it goes in

Reply:
@emilyjablonski ,
I think this can be simplified by splitting the date and time on `T`, and combining them.